app/views/homepages/index.html.erb

```diff
@@ -0,0 +1,75 @@
+<body>
+  <main>
+      <section class="spotlight">
+    <h2 class="spotlight__header">
+      <span class="spotlight__header--prefix">Media Spotlight</span>
+      <%= link_to @spotlight.title, works_path(@spotlight.id)%>
+      <small class="top-ten__creator">by <%=@spotlight.creator%></small>
+    </h2>
+    <p class="spotlight__description">
+      <%=@spotlight.votes.count%> vote(s)
+    </p>
+  </section>
+
+<hr class="root__hr">
+
+<section class="top-ten__container">
+  <section class="top-ten__list-container">
+  <h2 class="top-ten__header">
+    Top Movies
+  </h2>
+  <ul class="list-group top-ten__list">
```

Comment:
You essentially repeat the same code 3 times here. This is a great place to use a partial view to DRY up your code.